dotnet/src/Functions/Functions.OpenApi/Serialization/SimpleStyleParameterSerializer.cs

```diff
@@ -16,15 +16,15 @@ internal static class SimpleStyleParameterSerializer
     /// <param name="parameter">The REST API operation parameter to serialize.</param>
```

Comment:
Check any other `operation` lurking in comments 
```suggestion
    /// <param name="parameter">The REST API parameter to serialize.</param>
```

Reply:
Yep, thanks, done.